src/app/components/product-table/product-table.component.html

```diff
@@ -1,28 +1,55 @@
 <div class="product-table mat-elevation-z4">
+
+
   <div *ngIf="loading" class="loading-shade">
     <mat-spinner></mat-spinner>
   </div>
   <table mat-table matSort [dataSource]="dataSource">
     <ng-container matColumnDef="name">
       <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
-      <td mat-cell *matCellDef="let row">{{row.name}}</td>
+      <td mat-cell *matCellDef="let row">{{ row.name }}</td>
```

Comment:
Keep order for tag attributes: (change everywhere)
#id
*ngIf
mat-cell
aria-label="test"
[some]="some"
(click)="handler"
[(ngModule)]="item"